examples/02_decoding/plot_miyawaki_reconstruction.py

```diff
@@ -160,7 +160,7 @@ def flatten(list_of_2d_array):
         [
             ("selection", SelectKBest(f_classif, k=500)),
             ("scl", StandardScaler()),
-            ("clf", OMP(normalize=False, n_nonzero_coefs=10)),
+            ("clf", OMP(normalize="deprecated", n_nonzero_coefs=10)),
```

Comment:
normalize parameter can just be deleted no?

Reply:
yup that works: checked and no warning thrown